DefInjected/ThingDef/Races_Animal_Birds.xml

```diff
@@ -17,7 +17,7 @@
 	<Cassowary.lifeStages.0.label>птенец казуара</Cassowary.lifeStages.0.label>-->
 
 	<Emu.label>страус эму</Emu.label>
-	<Emu.description>Большая нелетающая птица с покрытыми капельками глазами на её ужасном лице. Дикое порождение природы.</Emu.description>
+	<Emu.description>Большая нелетающая птица с выпученными глазами. Дикое порождение природы.</Emu.description>
```

Comment:
_A large flightless bird with beady eyes on its ugly face. With its bad attitude, it is the jerk of the natural world._